python/py-tenacity/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+name                py-tenacity
+version             6.0.0
+platforms           darwin
+license             Apache-2
+maintainers         nomaintainer
+
+description         Retry code until it succeeds
+long_description    ${description}
+
+homepage            https://github.com/jd/tenacity
+master_sites        pypi:t/tenacity
+distname            tenacity-${version}
+
+checksums           rmd160  8e5f3e8813d5f39cd34cef1c46edc7490b74a037 \
+                    sha256  72f397c2bb1887e048726603f3f629ea16f88cb3e61e4ed3c57e98582b8e3571 \
+                    size    33589
+
+python.versions     27 35 36 37
```

Comment:
Please remove py27, 35 and 36. 

Why no 38 version? Without it, the other ports will not support a py38 sub-port.

Reply:
Thanks done, with `38`.